apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.sql

```diff
@@ -84,7 +83,6 @@ SELECT
 FROM
   assessments AS a
   JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
-  LEFT JOIN LATERAL authz_assessment (a.id, $authz_data, $req_date, ci.display_timezone) AS aa ON TRUE
```

Comment:
I think there's a reference to this in line 90...

Reply:
thanks, fixed